test/performance/run.sh

```diff
@@ -28,8 +28,8 @@ kubectl create clusterrole view-pods --verb=list,get --resource=pods,pods/status
 kubectl create clusterrolebinding view-pods --clusterrole view-pods --user system:serviceaccount:default:default --dry-run=client -o yaml | kubectl apply -f -
 export SA_TOKEN=$(kubectl create token default --namespace default)
 
-python3 -m venv venv
-./venv/bin/python -m pip install -r ${SCRIPT_DIR}/requirements.txt
+python3.11 -m venv venv
+./venv/bin/python3.11 -m pip install -r ${SCRIPT_DIR}/requirements.txt
```

Comment:
I don't have this version installed, if `python3` does not work for you change it to `python`. These days `python` is an alias for `python3` or whatever version you have installed.

Reply:
I'm sorry, I pushed this file by mistake...

Reply:
Removed the file from the commit.